go.mod

```diff
@@ -3,13 +3,17 @@ module github.com/ossf/allstar
 go 1.16
 
 require (
+	cloud.google.com/go v0.87.0 // indirect
 	github.com/bradleyfalzon/ghinstallation v1.1.1
-	github.com/google/go-github/v35 v35.2.0
-	github.com/google/go-querystring v1.1.0 // indirect
+	github.com/google/go-github/v29 v29.0.3 // indirect
+	github.com/google/go-github/v32 v32.1.0
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79
-	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
-	github.com/rs/zerolog v1.22.0 // indirect
-	gocloud.dev v0.23.0 // indirect
-	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
-	gopkg.in/yaml.v2 v2.2.8
+	github.com/ossf/scorecard v1.2.1-0.20210722153731-89c8e2af3131
```

Comment:
FYI after https://github.com/ossf/scorecard/pull/716 needs to be updated to `scorecard/v2`